src/Compilers/CSharp/Portable/Generated/Syntax.xml.Internal.Generated.cs

```diff
@@ -127,11 +129,8 @@ internal IdentifierNameSyntax(ObjectReader reader)
     {
       this.SlotCount = 1;
       var identifier = (SyntaxToken)reader.ReadValue();
-      if (identifier != null)
-      {
-         AdjustFlagsAndWidth(identifier);
-         this.identifier = identifier;
-      }
+      AdjustFlagsAndWidth(identifier);
```

Comment:
curious if there's any perf impact of this.  i.e. always having the method call where previously it would only call after a null check.  it's less IL and branching (it seems), but calling a method more often.  is it a wash?